tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/missing_concept_record_suppression_test.py

```diff
@@ -96,35 +121,37 @@ def test_missing_concept_record_suppression_cleaning(self):
                 -- Dropped due to unknown observation_concept_id --
                 (3, 3, 50000000, date('2020-05-05'), 441840, 441840, 441840, 441840, 441840, 441840),
                 -- Dropped due to unknown observation_type_concept_id --
-                (4, 4, 4297377, date('2020-05-05'), 19, 441840, 441840, 441840, 441840, 441840)
+                (4, 4, 4297377, date('2020-05-05'), 19, 441840, 441840, 441840, 441840, 441840),
+                -- NOT dropped. `_source_concept_id` columns can have unknown concepts --
+                (5, 5, 4225432, date('2020-05-05'), 441840, 441840, 441840, 441840, 50000000, 441840),
+                (6, 6, 4225432, date('2020-05-05'), 441840, 441840, 441840, 441840, 50000001, 441840)
             """).render(fq_dataset_name=self.fq_dataset_name)
 
-        queries.append(create_observations_query_tmpl)
+        queries = [insert_concept, insert_death, insert_aou_death, insert_obs]
 
         self.load_test_data(queries)
 
-        #Uncomment below and fill
-
         tables_and_counts = [{
-            'fq_table_name':
-                f'{self.fq_dataset_name}.observation',
-            'fq_sandbox_table_name':
-                self.fq_sandbox_table_names[0],
-            'loaded_ids': [1, 2, 3, 4],
+            'fq_table_name': f'{self.fq_dataset_name}.aou_death',
+            'fq_sandbox_table_name': self.fq_sandbox_table_names[0],
+            'loaded_ids': ['a', 'b', 'c', 'd', 'e'],
+            'sandboxed_ids': ['b', 'c'],
+            'fields': ['aou_death_id'],
+            'cleaned_values': [('a',), ('d',), ('e',)]
+        }, {
+            'fq_table_name': f'{self.fq_dataset_name}.death',
+            'fq_sandbox_table_name': self.fq_sandbox_table_names[1],
+            'loaded_ids': [1, 2, 3, 4, 5],
+            'sandboxed_ids': [2, 3],
+            'fields': ['person_id'],
+            'cleaned_values': [(1,), (4,), (5,)]
+        }, {
+            'fq_table_name': f'{self.fq_dataset_name}.observation',
+            'fq_sandbox_table_name': self.fq_sandbox_table_names[2],
+            'loaded_ids': [1, 2, 3, 4, 5, 6],
             'sandboxed_ids': [3, 4],
-            'fields': [
-                'observation_id', 'person_id', 'observation_concept_id',
-                'observation_date', 'observation_type_concept_id',
-                'value_as_concept_id', 'qualifier_concept_id',
-                'unit_concept_id', 'observation_source_concept_id',
-                'value_source_concept_id'
-            ],
-            'cleaned_values': [
-                (1, 1, 4225432, self.date, 441840, 441840, 441840, 441840,
-                 441840, 441840),
-                (2, 2, 4297377, self.date, 441840, 441840, 441840, 441840,
-                 441840, 441840),
-            ]
+            'fields': ['observation_id'],
+            'cleaned_values': [(1,), (2,), (5,), (6,)]
         }]
 
         self.default_test(tables_and_counts)
```

Comment:
```suggestion
        self.default_test(tables_and_counts)
        
```

Reply:
The irony is that you stripped out all that trailing whitespace, except the very [last line](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3#why-should-text-files-end-with-a-newline).

Reply:
I knew it is better to end with a new line, but I did not know why till right now! :sunglasses: